### PR TITLE
buffs milk for skeletons and plasmamen (not stolen)

### PR DESCRIPTION
**Pull request by JJRcop**
Steals @PKPenguin321's work

It's a little less embarrassing to set up travis for your own repo then wait for the build to complete :smile: 
Probably even better to just do it on your local repo but I digress

:cl: PKPenguin321 && !JJRcop
rscadd: Milk, which is good for your bones, is now extra beneficial to species that are mostly comprised of bones.
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Nice, thanks for figuring this out :+1:

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -135,6 +135,12 @@
 		. = 1
 	if(holder.has_reagent("capsaicin"))
 		holder.remove_reagent("capsaicin", 2)
+	var/datum/dna/Mdna = M.has_dna()
+	if(Mdna && Mdna.species)
+		var/datum/species/Mspecies
```
Comment (RemieRichards) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: nice null variable buddy.

Comment (duncathan) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Isn't species already typecast properly anyway

Comment (RemieRichards) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: y'know, it actually is.

Comment (JJRcop) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: oops

**Comment by JJRcop** [comment]
Oh right, it actually works now because of testing locally instead of risking it with the web editor

**Comment by PKPenguin321** [comment]
i feel your pain

**Pull request merged**